### PR TITLE
grabbing basic data from epic games

### LINKED ISSUE #33: Explore Epic scraping
## Description

Start scraping the Epic website, and explore the syntax used

## Required Files

N/A

## User Story

As an engineer, I need to be able to scrap the data from the website, so that it is in an adequate format to input into the database

### PR DESCRIPTION
**Pull request by shindym**
- Grabbing data required for database from epic games.

Closes #33

### CODE REVIEW

File: pipeline/extract_epic.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
+"""Extracting game details from EPIC games"""
+
+from os import environ
+from dotenv import load_dotenv
+import requests as req
+
```
Comment (annalisev) [inline pipeline/extract_epic.py:6]:
```suggestion
from dotenv import load_dotenv

import requests as req

```

**Review by hhm970 — APPROVED** [review]
Looks good!

File: pipeline/extract_epic.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+def get_game_details(game_obj):
+    """Returns list of relevant details from game json object."""
+    title = get_game_title(game_obj)
+    tags = get_tags(game_obj)
+    developer = get_developer(game_obj)
+    publisher = get_publisher(game_obj)
+    price = get_price(game_obj)
+    release_date = get_release_date(game_obj)
+    platform_ids = get_platform_ids(tags)
+    return [title, title, price, developer, publisher, release_date, tags, platform_ids]
```
Comment (annalisev) [inline pipeline/extract_epic.py:68]: Do we need two titles here? ```suggestion
    return [title, price, developer, publisher, release_date, tags, platform_ids]
```

**Review by annalisev — CHANGES_REQUESTED** [review]
A few small changes but otherwise perfect!

**Review by annalisev — APPROVED** [review]

**Review by Eibironke — APPROVED** [review]
👍

**Pull request merged**